### ISSUE TITLE
V.3.0.1 Paging not initializing correctly with simple TABLE

### ISSUE THREAD

**Issue by ghost** [fooplugins/FooTable#371]
I've built a simple, static HTML-Table following the docu, activated FooTable and paging for it with default settings but also tested several other settings step by step.

When I open the page I always see all rows of table. I have to click one pagination button after the other to hide them stepwise. Afterwards pagination works as expected.

I've built a minimal demo here:
http://re-later.eu/footable3-paging/footable3-paging.php

with test instructions here:
http://re-later.eu/footable3-paging/footable3-paging.php

I made several tests e.g. with TABLE attribute 

```
data-paging="true"
```

different other settings, window.load, tested several desktop browsers.

**Comment by ghost** [comment]
Maybe this issue is related https://github.com/fooplugins/FooTable/issues/333

**Comment by ghost** [comment]
Workaround to fix the problem:
Via JQuery:

```
<script type="text/javascript">
jQuery(function($){

 // Hide all TD before init of footable
 $("#tableselector tbody tr td").css("display", "none");

 $("#tableselector").footable(
 {
  "paging":{"enabled":true},
 }
 );
});
</script>
```

or via CSS

```
<style>
#tableselector tbody tr td{
 display:none;
}
</style>
```

**Comment by jhall39** [comment]
I have this same issue and the CSS works for me. I've been fighting this issue all morning thinking it was a problem with how I'm using it. How such a glaring bug ever made its way into this release, I'll never know. In my experiments I noticed that FooTable would add on the paging rows to the bottom of my table. So I would see all several hundred rows of my table, plus the 3 or so of FooTable's current page. Those rows would change with the page selection while leaving the rest of my table alone. Thanks to bertmert's CSS above, it's working right fine now.

**Comment by steveush** [comment]
Hi, this has been fixed in 3.0.2.

**Comment by ghost** [comment]
Thank you!
